precode.sh

```diff
@@ -6,38 +6,40 @@
 #   dir2
 #   dir3
 #       dir4
-
+mkdir -p ./task ./task/dir1 ./task/dir2 ./task/dir3 ./task/dir3/dir4
 # изменяем текущую директорию на task
-
+cd ./task
 # создаём пустой файл task/dir2/empty
-
+touch ./dir2/empty
 # создаём файл task/dir2/hello.sh с таким содержанием:
 # #!/bin/bash
 # echo "$1, привет!"
-
+echo '#! /bin/bash
```

Comment:
Тут с пробелом не будет работать
```suggestion
echo '#!/bin/bash
```